examples/storage_volume_attachment.go

```diff
@@ -32,7 +33,12 @@ func main() {
 	for i := 0; i < len(attachment_list.Members); i++ {
 		fmt.Println(attachment_list.Members[i].URI)
 	}
+	id := filepath.Base(string(attachment_list.Members[0].URI))
 
+	// Get volume attachment by id
+	fmt.Println("\nGetting details of volume attachment with name: ", id)
+	volAttach_by_id, _ := ovc.GetStorageAttachmentById(id)
+	fmt.Println(volAttach_by_id)
```

Comment:
Please add the if condition for err response.

Reply:
Done